dev-packages/browser-integration-tests/suites/tracing/browserTracingIntegration/pageload/test.ts

```diff
@@ -1,10 +1,16 @@
 import { expect } from '@playwright/test';
 import type { Event } from '@sentry/types';
 
+import {
+  SEMANTIC_ATTRIBUTE_SENTRY_OP,
+  SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN,
+  SEMANTIC_ATTRIBUTE_SENTRY_SAMPLE_RATE,
+  SEMANTIC_ATTRIBUTE_SENTRY_SOURCE,
+} from '@sentry/browser';
 import { sentryTest } from '../../../../utils/fixtures';
 import { getFirstSentryEnvelopeRequest, shouldSkipTracingTest } from '../../../../utils/helpers';
 
-sentryTest('should create a pageload transaction', async ({ getLocalTestPath, page }) => {
+sentryTest('creates a pageload transaction with url as source', async ({ getLocalTestPath, page }) => {
```

Comment:
no behaviour change here, just to explicitly check that our default browserTracingIntegration starts spans with source "url"